microcosm_pubsub/daemon.py

```diff
@@ -64,15 +62,10 @@ def handler(event, context):
             if "warm" in event:
                 return "warming up"
 
-            message = event["Records"][0]["body"]
```

Comment:
I don't see this change in event contract reflected in the test. Test modified doesn't seem to have `Records`.

Reply:
you right

Reply:
This is true. The test does not have Records because it's testing only the consumer itself.